.github/workflows/pull-request.yml

```
@@ -42,11 +42,11 @@
 
       - name: Get changed TypeScript files
         id: changed-files
-        uses: tj-actions/changed-files@v40
+        uses: tj-actions/changed-files@v45
       - name: Check formatting
         if: steps.changed-files.outputs.only_changed != 'true'
         run: npm run format:check
-
+        
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)